packages/python-sdk/e2b/session/process.py

```diff
@@ -208,13 +209,14 @@ async def start(
         :param on_stderr: A callback that is called when stderr with a newline is received from the process
         :param on_exit: A callback that is called when the process exits
         :param env_vars: A dictionary of environment variables to set for the process
-        :param rootdir: The root directory for the process
+        :param cwd: The root directory for the process
+        :param rootdir: (DEPRECATED - use cwd) The root directory for the process
```

Comment:
We could add the deprecated directive, even though we aren't using sphinx atm
[deprecated directive](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated)
```suggestion
        :param rootdir: (DEPRECATED - use cwd) The root directory for the process
        .. deprecated:: 0.3.2
            Use cwd instead.
        
```